src/plugin/heatmap/heatmapsynchronizertype.js

```diff
@@ -0,0 +1,8 @@
+goog.module('plugin.heatmap.SynchronizerType');
```

Comment:
Nit: looks like multiple provides got split into multiple files that could be consolidated into a general `plugin.heatmap` module.

Reply:
Yeah the script attempts to split out the provides in a way that will not be a breaking change. If something has a specific `provide`, we want `goog.require` to continue to work for that same name so a refactor like this would constitute a breaking change.